test/fixtures/files/registrar_data_full_anonymized.csv

```diff
@@ -1,7 +1,7 @@
 Full Name,Degree Status,Degree Type,Degree Desc,Degree Department,Degree Award Term,Degree Award Date,Degree Code,Dept Name In Commencement Bk,Thesis Title,Is Primary Course Of Major,Thesis Coauthor,Krb Name,Last Name,First Name,Middle Name,Email Address
 "Klein, Jennifer",AW,PHD,Doctor of Philosophy,MAS,2019SU,9/18/2019 0:00,PHD,Program in Media Arts and Sciences,thesis in the field of Media Arts and Sciences: Social Inductive Biases for Reinforcement Learning,Y,,finleyjessica,Klein,Jennifer,,finleyjessica@example.com
-"Hawkins, Destiny",AW,PHD,Doctor of Philosophy,MAS,2019SU,9/18/2019 0:00,PHD,Program in Media Arts and Sciences,thesis in the field of Media Arts and Sciences: Laser Direct-Write Fabrication of MEMS,Y,,pattersondavid,Hawkins,Destiny,,pattersondavid@example.com
-"Love, Tyler Alexander",AW,PHD,Doctor of Philosophy,6,2019SU,9/18/2019 0:00,PHD,Department of Electrical Engineering and Computer Science,thesis in the field of Computer Science: Fast Spectral Primitives for Directed Graphs,Y,,dianebishop,Love,Tyler,Alexander,dianebishop@example.com
+"Hawkins, Destiny",AW,PHD,Doctor of Philosophy,MAS,2019SU,5/18/2023 0:00,PHD,Program in Media Arts and Sciences,thesis in the field of Media Arts and Sciences: Laser Direct-Write Fabrication of MEMS,Y,,pattersondavid,Hawkins,Destiny,,pattersondavid@example.com

```

Comment:
Was there a reason you changed these fixture dates? Is it likely we'll need to keep increasing them over time or will they be stable with these years? I think it's just because you needed some new degree periods to be detected but I want to make sure these weren't changing for some side effect reason I'm not following :)

Reply:
@JPrevost Sorry, should have mentioned this in the side effects. If I didn't update this fixture, we would have had only one new degree date, since all of the theses in the CSV have the same degree date. One updated record felt dangerous to confirm that it's working properly, so I changed a couple of rows so we would at least have three new degree dates.